src/index.html

```diff
@@ -20,8 +20,271 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link">apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the link
```suggestion
          </li>
          
          <li class="nav__item">
```